spec/integration/manifest.spec.yml

```diff
@@ -233,6 +233,14 @@ spec:
                   - organization
                   - division
                   - team
+    required_vars:
```

Comment:
We should add a version patch below under `versions:`, it would be something like:
```
  - before: 3.3.2
    patch:
      - op: remove
        path: "/properties/policy_templates/items/properties/required_vars"
```